### ISSUE TITLE
Xenoblade Chronicles 60FPS

### ISSUE THREAD

**Issue by homebrewGT** [libretro/dolphin#237]
I tried to use the Gecko code to make Xenoblade Chronicles 60fps but i haven't had any luck, how to do this?

**Comment by bslenul** [comment]
This place isn't for support, please use Reddit, Discord or the forums next time ;)

But anyway:
* 1st make sure to turn ON internal cheats from `Quick Menu > Options`, then close RA.
* Download that file (which I've created based on [the Dolphin wiki](https://wiki.dolphin-emu.org/index.php?title=Xenoblade_Chronicles#60_FPS_unlock)): [SX4.zip](https://github.com/libretro/dolphin/files/7409875/SX4.zip)
* Extract the .ini file in your `retroarch/saves/User/GameSettings/` folder.
 
Then launch the game again and it *should* work :)

Before:
![30](https://user-images.githubusercontent.com/33353403/138689979-0cf3b236-4b4b-428f-8244-c011d6ce3cce.png)

After:
![60](https://user-images.githubusercontent.com/33353403/138690021-1383b6ca-5069-4573-984c-1c456b5edb88.png)

**Comment by homebrewGT** [comment]
Unfortunately it didnt work, i have internal cheats enabled and the file in the right place. Should i rename the file to the game's name extension?

**Comment by bslenul** [comment]
No, the .ini is named after the game ID, so SX4.ini in this case. This is the 60fps code for the US tho, is that the version you're using?

**Comment by homebrewGT** [comment]
Yes i am using the US version, which is SX4E01.ini, I'm using RetroArch Dolphin core on Xbox Series X, the Twilight Princes Hack worked but this one doesnt seem to work. I'm not sure what is wrong.

EDIT:
And i tried enabling other cheats just to test but it seems like it wont recognize any of them..

**Comment by bslenul** [comment]
Erm, not sure then :/ Works fine on Windows but I don't have an Xbox to test, although I don't think the platform should matter anyway.

**Comment by homebrewGT** [comment]
Thanks for the help anyway, I know the core is recognizing the cheat file because it fixed the "save screenshot" thing but no luck with any of the Gecko Codes for it.

**Comment by Sanaki** [comment]
Just tested this and encountered the same issue. The issue he's having is a symptom of something else where the config is somehow resetting and not honoring specific core options properly. As a result, internal cheats just aren't enabled at all. Ended up just outright nuking the Config directory to get it to work, though I'm not even sure that was what finally did it. Some settings reset themselves over and over regardless of what you do. It's like there's another config it's reading somewhere, until some bit of random magic causes that one to update properly.

Unfortunately, this also means per-game configs are effectively broken for any settings that can't change function on the fly after initial core load. I noticed this previously with differing widescreen settings and couldn't pin down what was causing it. Now I know.
